index.html

```diff
@@ -273,7 +285,8 @@ <h2>Skills</h2>
                 <form action="https://formspree.io/f/xyyaopza" method="POST" class="contact-form">
                     <input name="FullName" type="text" placeholder="Full name" required maxlength="30" />
                     <input name="Email" type="email" placeholder="Email address" required />
-                    <textarea name="Message" rows="4" cols="50" placeholder="Write me something..." maxlength="500" required></textarea>
+                    <textarea name="Message" rows="4" cols="50" placeholder="Write me something..." maxlength="500"
+                        required></textarea>
                     <div class="contact-button">
                         <button type="submit">Get in touch</button>
```

Comment:
- [ ] Kindly look into the two screenshots I presented below with marks when changes are necessary and walk you design alongside the Figma design

### Your Design
![your-form](https://user-images.githubusercontent.com/26597432/236642978-6a58ca90-a067-44fe-a421-9d0c5de42ce4.png)

### Figma Design
![tem-form](https://user-images.githubusercontent.com/26597432/236642990-25a801fb-ae55-477d-9131-e2b2ab168b0d.png)